website/docs/d/is_vpn_server_route.html.markdown

```diff
@@ -23,31 +23,22 @@ data "ibm_is_vpn_server_route" "example" {
 
 Review the argument reference that you can specify for your data source.
 
-- `id` - (Required, String) The VPN route identifier.
+- `identifier` - (Optional, String) The VPN route identifier.
+- `name` - (Optional, String) The VPN route identifier.
 - `vpn_server` - (Required, String) The VPN server identifier.
 
+  ~> **NOTE:** `identifier` and `name` are mutually exclusive.
+
 ## Attribute Reference
 
 In addition to all argument references listed, you can access the following attribute references after your data source is created.
 
 - `id` - The unique identifier of the VPNServerRoute.
```

Comment:
id format is vpnserverID/vpnserverRouteID. we can mention that. same in vpnserver route resource as well